TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/common/sockets/MessageTypeNotFoundExcpetion.java

```diff
@@ -0,0 +1,9 @@
+package cloud.timo.TimoCloud.common.sockets;
+
+import cloud.timo.TimoCloud.common.protocol.MessageType;
+
+public class MessageTypeNotFoundExcpetion extends RuntimeException{
+    public MessageTypeNotFoundExcpetion(MessageType messageType) {
+        super("Could not categorize message type: " + messageType);
```

Comment:
Change the message to `"Could not find any message handlers for the message type: " + messageType`